core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsFeed.kt

```diff
@@ -57,7 +57,11 @@ fun LazyGridScope.newsFeed(
     when (feedState) {
         NewsFeedUiState.Loading -> Unit
         is NewsFeedUiState.Success -> {
-            items(feedState.feed, key = { it.id }) { userNewsResource ->
+            items(
+                feedState.feed,
```

Comment:
nit: use named argument for consistency
```suggestion
                items = feedState.feed,
```

Reply:
I realise this wasn't your code but might as well fix it while you're here :)